example/image-classification/predict-cpp/image-classification-predict.cc

```diff
@@ -1,283 +1,298 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
 /*!
- *  Copyright (c) 2015 by Xiao Liu, pertusa, caprice-j
```

Comment:
I don't think you can do this, can you? I was under the impression that license text could never be removed.
  @smarthi , this this ok?

Reply:
Technically I'm not removing contributors, just naming them as a group. I followed the pattern in the rest of the project, but will be happy to change it back.

Reply:
is the guy’s name still there?

Reply:
Yes, brought it back